### ISSUE TITLE
Meeting Notes Cleanup

### ISSUE THREAD

**Issue by whoabuddy** [stacksgov/pm#136]
Rather than track missing notes for each call in its own issue, I'm grouping everything together so its easier to know what needs to be cleaned up.

Shout out to @brexton017 for all the help so far, this list is getting smaller!

- [ ] Call 1 needs notes
- [ ] Call 2 needs notes, add Lane's comments from GH issue
- [ ] Call 3 has extra notes to add (from issue)
- [ ] Call 13 needs notes
- [ ] Call 16 needs reformatting (no line breaks)
- [ ] Call 18 needs notes

From Call 21 forward we are uploading the raw transcription from Zoom, which serves as a starting point for the notes and usually needs some cleanup. Once the list above is complete we can start finalizing the process for new notes moving forward.

**Comment by whoabuddy** [comment]
In #167 there was consensus to continue and catch up the notes through the current call. This will require some coordination between myself, @jennymith and @HaroldDavis3 to get the recordings from Zoom.

**Comment by whoabuddy** [comment]
Linking this pull request and will work to get the table updated so at least we know what we're targeting!

https://github.com/stacksgov/resources/pull/54

**Comment by HaroldDavis3** [comment]
> In #167 there was consensus to continue and catch up the notes through the current call. This will require some coordination between myself, @jennymith and @HaroldDavis3 to get the recordings from Zoom.

@whoabuddy If it is ok to work our way backwards, can we start with some recordings I have so far? Believe Jenny's gonna need some more time. Just lmk best way to share if we can do it.
